src/mumble/ListenerLocalVolumeSlider.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_LISTENERLOCALVOLUMESLIDER_H_
+#define MUMBLE_MUMBLE_LISTENERLOCALVOLUMESLIDER_H_
+
+class Channel;
+
+#include "VolumeSliderWidgetAction.h"
+
+class ListenerLocalVolumeSlider : public VolumeSliderWidgetAction {
+	Q_OBJECT
+
+	/// The channel of the listener proxy this dialog is operating on
+	const Channel *m_channel;
```

Comment:
private and/or protected members should be listed after the `public` ones - most of the time the reader will only be interested in  the public interface of a class, so it makes sense if they find that first.